build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+sudo apt-get update
+sudo apt-get install -y libfuse2 ffmpeg pkg-config
```

Comment:
1. why do we need `libfuse2`?
2. why do we need ffmpeg if we are linking against it statically?

Reply:
Removed and verified the build succeeds and still works

Reply:
Removed "seanorg" stuff reverting back to erh/viamrtsp for everything. Should be good to merge if no more suggestions @nicksanford